### PR TITLE
VAE notebook

### PR DESCRIPTION
**Pull request by rijobro**
VAE example notebook. Will require https://github.com/Project-MONAI/MONAI/pull/1148.

### CODE REVIEW

File: modules/varautoencoder_mednist.ipynb [hunk 1]
```diff
@@ -0,0 +1,594 @@
+{
```
Comment (wyli) [inline modules/varautoencoder_mednist.ipynb:1]: <p>it seems <code>loss_balance = []</code>  is unused and could be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (wyli) [inline modules/varautoencoder_mednist.ipynb:1]: <p>this visualisation could be done on the latent coordinates generated by the inverse CDF (to be consistent with https://arxiv.org/pdf/1312.6114.pdf appendix A figure 4)</p><pre class="ql-syntax" spellcheck="false">%matplotlib inline
num_ims=10
pl.figure(figsize=(20,12))
out=[[list() for _ in range(num_ims)] for _ in range(latent_size-1)]
dist = torch.distributions.normal.Normal(torch.tensor(0.), torch.tensor(1.))
model.eval()
with torch.no_grad():
    for z in range(latent_size-1):
        for z in range(latent_size-1):
            for y,j in enumerate(torch.linspace(0.05,0.95,num_ims)):
                for x,i in enumerate(torch.linspace(0.05,0.95,num_ims)):
                    sample = torch.zeros(1,latent_size).to(device)
                    sample[0,z] = dist.icdf(j)
                    sample[0,z+1] = dist.icdf(i)
                    o=model.decode_forward(sample)
                    o=o.detach().cpu().numpy().reshape(im_shape[1:])
                    out[z][y].append(o)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

**Review by wyli — APPROVED** [review]
